test/server/rpc/jsdoc-extractor.spec.js

```diff
@@ -0,0 +1,88 @@
+const assert = require('assert'),
+    utils = require('../../assets/utils.js'),
+    jp = utils.reqSrc('rpc/jsdoc-extractor.js');
+
+describe('jsdoc-extractor', () => {
+
+    let comment = `
+    /**
+     * this is the description
+     * next line of description
+     * @param {string} address target address
+     * @param {number} limit the results limit
+     * @param {Object} options
```

Comment:
A minor point but it would be nice to follow one convention for the types everywhere (even in test cases). Specifically, whether or not we are capitalizing the first word of the type like `Number` (the fn in JavaScript) or `number` (the output of `typeof 5` in JavaScript).